src/core/server/saved_objects/migrations/model/retry_state.ts

```diff
@@ -10,20 +10,27 @@ import { State } from '../state';
 
 export const delayRetryState = <S extends State>(
   state: S,
-  errorMessage: string,
+  errorMessage: string, // comes in as "[index_not_yellow_timeout] Timeout waiting for the status of the [${index}] index to become 'yellow'"
```

Comment:
```suggestion
  errorMessage: string,
```

When it's a retryable es error there will be a different errorMessage

Reply:
oops, great catch!